bindings/python/src/OpenSpaceToolkitMathematicsPy/Geometry/2D.cpp

```diff
@@ -13,10 +12,8 @@ inline void OpenSpaceToolkitMathematicsPy_Geometry_2D(pybind11::module& aModule)
     // Add __path__ attribute for "d2" submodule
     d2.attr("__path__") = "ostk.mathematics.geometry.d2";
 
-    // Add objects to python "d2" submodules
-    OpenSpaceToolkitMathematicsPy_Geometry_2D_Object(d2
-    );  // Cannot be binded without including Transformation.cpp (uses forward declaration for Transformation)
-    OpenSpaceToolkitMathematicsPy_Geometry_2D_Objects(d2);
```

Comment:
This is no longer required because instead of having `d2::Object` and `d2::objects::...` they now share the singular `object` name and therefore share a pybind file.